packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -415,8 +415,20 @@ class _RefreshProgressIndicatorPainter extends _CircularProgressIndicatorPainter
   }
 }
 
-
+/// A progress indicator for refreshing the contents of a widget.
```

Comment:
well the progress indicator itself doesn't refresh the contents... For notifying the user that the contents are being refreshed, maybe?